src/esp/gfx_batch/RendererStandalone.cpp

```diff
@@ -164,13 +165,42 @@ Mn::Image2D RendererStandalone::colorImage() {
                                   colorFramebufferFormat());
 }
 
+void RendererStandalone::colorImageInto(const Magnum::Range2Di& rectangle,
+                                        const Mn::MutableImageView2D& image) {
```

Comment:
Nitpick: I would rename `colorImageInto` and `depthImageInto` to something a bit more explicit.